packages/common/test/events/HubEvent.test.ts

```diff
@@ -90,9 +90,11 @@ describe("HubEvent Class:", () => {
   });
 
   it("delete", async () => {
-    const deleteSpy = spyOn(eventModule, "deleteEvent").and.callFake(() => {
-      return Promise.resolve();
-    });
+    const deleteSpy = spyOn(eventEditModule, "deleteHubEvent").and.callFake(
+      () => {
+        return Promise.resolve();
+      }
+    );
     const chk = HubEvent.fromJson({ name: "Test Event" }, authdCtxMgr.context);
     await chk.delete();
```

Comment:
The way this test is written will result in the `expect` statements not being invoked within the `catch` blocks if `chk.delete` succeeds for some reason. We should add call to `fail` after we invoke `chk.delete` and `chk.save` to make sure the test fails if those operations for some reason succeed in the future.

```ts
    try {
      await chk.delete();
      fail('delete did not reject');
    } catch (e) {
      expect(e.message).toEqual("HubEvent is already destroyed.");
    }

    try {
      await chk.save();
      fail('save did not reject');
    } catch (e) {
      expect(e.message).toEqual("HubEvent is already destroyed.");
    }
```